src/app/(auth)/onboarding/page.tsx

```diff
@@ -4,6 +4,7 @@ import OnboardingForm from "@/components/shared/Form/Onboarding/OnboardingForm";
 import { getSession } from "~/auth/auth";
 import { redirect } from "next/navigation";
 
+// { [key: string]: string | string[] | undefined }
 const Onboarding = async ({ searchParams }: any) => {
```

Comment:
```suggestion
const Onboarding = async ({
  searchParams,
}: {
  searchParams: { step: number };
}) => {
```
I believe that comment that was added here was for the types on your search params right? Let's just remove the comment and add the type to the search params instead of leaving a comment like that and getting it merged into main 👍